src/lib.rs

```diff
@@ -88,8 +91,8 @@ pub struct Entity {
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize, Debug))]
 #[derive(Encode, Decode, Default, Clone, PartialEq, Eq)]
 pub struct ClassSchema {
-    version: u16,
-    properties: Vec<u8> // indices into properties vector for the corresponding class
+    /// Indices into properties vector for the corresponding class.
```

Comment:
> 

> We could later switch out both `properties` and `schemas` in `Class` from `Vec<_>` containers to `BTreeMap`

Unfortunately, Jaco from Parity (he works on Polkadot UI) said that there is no support for `BTreeMap` in Substrate codec because they don't think that `BTreeMap` is something that should be widely used in storage values.